pulsar-transformations/src/test/java/com/datastax/oss/pulsar/functions/transforms/FlattenStepTest.java

```diff
@@ -264,21 +263,21 @@ void testNestedKeyValueNonAvroSchema() throws Exception {
     Utils.process(nestedKVRecord, FlattenStep.builder().part("value").build());
   }
 
-  @Test(
-    expectedExceptions = IllegalStateException.class,
-    expectedExceptionsMessageRegExp = "Flatten requires non-null schemas!"
-  )
-  void testNestedSchemalessValue() throws Exception {
-    // given
-    Record<GenericObject> nestedKVRecord =
-        new Utils.TestRecord<>(
-            null,
-            AutoConsumeSchema.wrapPrimitiveObject("value", SchemaType.STRING, new byte[] {}),
-            "myKey");
-
-    // then
-    Utils.process(nestedKVRecord, FlattenStep.builder().part("value").build());
-  }
+//    @Test(
+//      expectedExceptions = IllegalStateException.class,
+//      expectedExceptionsMessageRegExp = "Flatten requires non-null schemas!"
+//    )
+//    void testNestedSchemalessValue() throws Exception {
```

Comment:
Can a received message schema be null ?

Reply:
If you are using AUTO_CONSUME then it should not possibile, but I am not sure.